apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-content.js

```diff
@@ -74,10 +77,26 @@ function getTourContent() {
 			animation: 'undo-button',
 		},
 		{
-			heading: __( 'Congratulations!', 'full-site-editing' ),
-			description: __(
-				'You’ve now learned the basics. Remember, your site is always private until you decide to launch.',
-				'full-site-editing'
+			heading: __( 'Congratulations!', 'full-site-editing' ).replace( '&nbsp;', ' ' ),
+			description: createInterpolateElement(
+				__(
+					'You’ve now learned the basics. Remember, your site is <link_to_launch_site_docs>always private until you decide to launch</link_to_launch_site_docs>. Check out the docs to <link_to_editor_docs>learn more about Block Editing</link_to_editor_docs>.',
```

Comment:
@eru One last tweak:  Could you update the text to Ollie's suggestion: https://github.com/Automattic/wp-calypso/pull/52099#issuecomment-824453811

Thank you. 😄

Reply:
@autumnfjeld Hmm, looks like I have the wrong account to reply to. 🤣

Reply:
Oh! I'm so sorry (my team mate is named Eru 😃 .   @roo2 this one's for you of course.  

 One last tweak: Could you update the text to Ollie's suggestion: https://github.com/Automattic/wp-calypso/pull/52099#issuecomment-824453811